containers/kubernetes-pod.json

```diff
@@ -6,30 +6,34 @@
          "kind":"ReplicationController",
          "apiVersion":"v1",
          "metadata":{
-            "name":"cockpit-kube",
```

Comment:
I'm not sure about renaming this file, we call it pod, but include a service definition. Same goes for all the docs.